src/schema/feeds.ts

```diff
@@ -212,6 +212,11 @@ export const typeDefs = /* GraphQL */ `
       """
       unreadOnly: Boolean = false
 
+      """
+      Force refresh the feed
+      """
+      refresh: Boolean = false
```

Comment:
As we synced on slack, we should add a metric to keep this usage in check. Ideally we would have a grace period internally in feed service to avoid cache overrides to often.